bundle/android/strings.xml

```diff
@@ -160,6 +161,7 @@
     <string name="gutenberg_native_edit_media" tools:ignore="UnusedResources">Edit media</string>
     <string name="gutenberg_native_edit_using_web_editor" tools:ignore="UnusedResources">Edit using web editor</string>
     <string name="gutenberg_native_edit_video" tools:ignore="UnusedResources">Edit video</string>
+    <string name="gutenberg_native_edit_video_024aee6d" tools:ignore="UnusedResources">Edit video</string>
```

Comment:
The following strings are duplicated on Android, however, they are not on iOS. Seems that form some reason, the i18n update scripts are identifying these strings as new. I'd like to explore this further but in the meantime, I wouldn't block the PR due to this.

- `Edit video`
- `Video caption. Empty`
- `Video caption. %s`

Reply:
One option I'm considering is that since they are associated with the Jetpack plugin, they should be considered different from the ones provided in Gutenberg. However, I noticed that something similar is happening on the Jetpack VideoPress plugin ([example](https://github.com/Automattic/jetpack/blob/f1ae7fb120221b3c678e11f4008149ab04abf289/projects/packages/videopress/src/client/block-editor/blocks/video/components/replace-control/index.native.js#L46)) and they weren't marked as new 🤔.

Reply:
Thank you for flagging this, Carlos. I'm going to go ahead to begin the merge domino for this PR, and it's associated changes as you mentioned this is non-blocking. But, will be happy to help look further into this specific issue with you next week.